composer.json

```diff
@@ -15,6 +15,12 @@
     },
     "require": {
         "php": ">=5.4",
+        "ext-pdo_mysql": "*",
+        "ext-mcrypt": "*",
+        "ext-intl": "*",
+        "ext-gd": "*",
```

Comment:
Yes, you can use gd / imagick or gmagick but in composer you can not have conditionnal requirements, so I think you have to remove this requirement.
But it would be great to add this check in the CheckPermission class.